data/moves.json

```diff
@@ -0,0 +1,974 @@
+[{}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {
```

Comment:
The empty objects are so that I can access the array via the ID in a forEach, and each Move needs to be in its correct spot.